pages/linux/ldapdomaindump.md

```diff
@@ -0,0 +1,20 @@
+# ldapdomaindump
+
+> Dumps users/computers/groups and OS/membership information via LDAP to HTML/JSON/greppable output.
```

Comment:
```suggestion
> Dumps information in formats of `users/computers/groups` and `OS/membership` via LDAP to HTML, JSON and greppable output.
```

Reply:
These are not formats !
I guess I could improve my sentence the following way : 
`Dumps users, computers, groups, OS and membership information via LDAP to HTML, JSON and greppable output.`

Reply:
Sure, go for it.